pytorch_lightning/accelerators/accelerator.py

```diff
@@ -59,9 +59,9 @@ def barrier(self, name: Optional[str] = None):
     def broadcast(self, obj, src=0):
         return obj
 
-    def train_or_test(self):
-        if self.trainer.testing:
-            results = self.trainer.run_test()
+    def train_or_evaluate(self):
```

Comment:
The name here is a bit misleading as this also runs test

Reply:
I used `evaluate` here to refer to either `test` or `validate`. I think it was inspired by the pre-existing `Trainer.run_evaluation` method, which is used to run either the test or validation loop depending on the value of the `test_mode` parameter.

Let me know if you have a better idea for the name!